### DISCUSSION TITLE
any tutorial on how to detach the database and put it in separate server?

### DISCUSSION THREAD

**Discussion by abcnow** [serverpod/serverpod#786] [Q&A] [answered]
I wonder if there is a tutorial on how to detach the database and put it on a different server... what would be the best strategy to follow here? I am using digitalocean.. The idea is to have database in a different server and use their Spaces (s3 version) for static content like images and videos..

**Comment by vlidholt** [comment, marked as answer]
You can connect ot any Postgres instance from your Serverpod, the only thing you need to do is to edit the configuration file and the passwords file. You can find the configuration files in the server's `config/` directory.
